README.md

```diff
@@ -62,11 +62,191 @@ httparty "https://api.stackexchange.com/2.2/questions?site=stackoverflow"
 
 ## Help and Docs
 
-* [Docs](https://github.com/jnunemaker/httparty/tree/master/docs)
-* https://groups.google.com/forum/#!forum/httparty-gem
 * https://www.rubydoc.info/github/jnunemaker/httparty
 * http://stackoverflow.com/questions/tagged/httparty
 
+
+### Table of contents
+- [Options](#options)
+- [Parsing JSON](#parsing-json)
+- [Working with SSL](#working-with-ssl)
+
+### Options
+
+HTTParty provides lot's of options for customization
+
+##### Request Options
+
+- :body - Body of the request. If passed an object that responds to #to_hash, will try to normalize it first, by default passing it to ActiveSupport::to_params. Any other kind of object will get used as-is.
```

Comment:
I'd probably wrap each of these in backticks so they show up more like code, but not a blocker.